network-config-analyzer/CalicoPolicyYamlParser.py

```diff
@@ -549,7 +549,7 @@ def _set_affects_ingress_egress(self, policy_spec, is_profile, res_policy):
         :return: None
         """
         allowed_policy_keys = {'order': 0, 'selector': 0, 'ingress': 0, 'egress': 0, 'types': 0, 'labelsToApply': 0,
-                               'doNotTrack': 3, 'preDNAT': 3, 'applyOnForward': 3}
+                               'namespaceSelector': 0, 'doNotTrack': 3, 'preDNAT': 3, 'applyOnForward': 3}
```

Comment:
Maybe add an argument to this method to indicate if the policy is GlobalNetworkPolicy or not ,and only add the `namespaceSelector` to the `allowed_policy_keys` if it is global.